roles/debian_backports/vars/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+backports_distribution: "{{ansible_distribution_release}}"
```

Comment:
I think to follow normal convention you'd want these to be 'debian_backports` to align with the role name.

Reply:
You're right, I just copied and made minor style changes to this module but sounds like a good modification for consistency.